openlibrary/macros/DonateModal.html

```diff
@@ -32,7 +32,7 @@ <h2 class="donation__override">
       <h2>Benefits of donating</h2>
       <p>When you donate a physical book to the Internet Archive, your book will enjoy:</p>
       <div class="donation__info">
-        <img src="$book.get_cover_url('M')">
+        <img src="$book.get_cover_url('M')" alt="Donation info">
```

Comment:
ideally this would  be translated but I understand there is a greater effort going there, so am focusing on accessibility first :)

Reply:
(As a side note, all we need to make these translateable, is wrap them with the `_` function like so: `alt="$_('Donation info')"`)